angr/analyses/calling_convention.py

```diff
@@ -295,6 +295,17 @@ def _generate_callsite_subgraph(self, func: 'Function', callsite_block_addr: int
         for _, dst, data in func.graph.out_edges(the_block, data=True):
             subgraph.add_edge(the_block, dst, **data)
 
+            # If the target block contains only direct jump statements and has only one successor,
+            # include its successor.
+
+            # Re-lift the target block
+            dst_bb = self.project.factory.block(dst.addr, func.get_block(dst.addr).size, opt_level=1)
+
+            # If there is only one 'IMark' statement in vex --> the target block contains only direct jump
+            if len(dst_bb.vex.statements) == 1 and len(func.graph.out_edges(dst)) == 1:
```

Comment:
Can you explicitly check dst_bb.vex.statements[0] to ensure that it's an IMark statement (instead of something else)?

Reply:
Also, `len(func.graph.out_edges(dst))` should be replaced by `func.graph.out_degree(dst)`.